tensorflow/python/BUILD

```diff
@@ -1926,6 +1926,8 @@ tf_cuda_library(
 tf_py_wrap_cc(
     name = "pywrap_tensorflow",
     srcs = ["tensorflow.i"],
+    # set linkstatic = 0 so that only apply whole archive to libraries that need to be force linked.
```

Comment:
Can you clarify this comment? (Perhaps mentioning that it (presumably?) is only necessary for Windows.)